tensordict/tensordict.py

```diff
@@ -4391,6 +4373,27 @@ def set(
         self._save(td)
         return self
 
+    def set_default(
+        self, key: NESTED_KEY, item: COMPATIBLE_TYPES, inplace: bool = False, **kwargs
+    ) -> COMPATIBLE_TYPES:
+        """Returns the value of the key if the key is in the tensordict. If not, insert
```

Comment:
ideally the first line should not be split, ie

"""Returns the value of the key if the key is in the tensordict. If not, insert key with a value of item and returns item.

Reply:
I guess it should be `inserts` with an s?

Reply:
Ah yes, it does read a bit funny. I just copied it from `set_default`, but let me take a pass at rewriting it.

Reply:
Have amended the docstring, taking inspiration from `dict.setdefault`.